charts/rstudio-launcher-rbac/README.md

```diff
@@ -55,4 +55,3 @@ helm template -n rstudio rstudio-launcher-rbac rstudio/rstudio-launcher-rbac
 
 ----------------------------------------------
 Autogenerated from chart metadata using [helm-docs v1.5.0](https://github.com/norwoodj/helm-docs/releases/v1.5.0)
-
```

Comment:
It doesn't really show well on github, but there were 2 newlines at the end of this file and we removed 1 of them.